app/helpers/admin_helper.rb

```diff
@@ -1,35 +1,37 @@
 module AdminHelper
   
   def get_memento_uri_from_handler(handler)
```

Comment:
Really this feels like another class rather than included in helpers, or is there a way to provide a relation between the`Job` and `Memento` may make this easier.